src/python/pants/core/util_rules/partitions.py

```diff
@@ -36,7 +36,31 @@ class PartitionerType(Enum):
     """The plugin author has a rule to go from `RequestType.PartitionRequest` -> `Partitions`."""
 
     DEFAULT_SINGLE_PARTITION = "default_single_partition"
-    """Registers a partitioner which returns the inputs as a single partition."""
+    """Registers a partitioner which returns the inputs as a single partition
+
+    The returned partition will have no metadata."""
```

Comment:
Can you move these trailing quotes to a newline? Makes it edit-friendly (less noise in the diff)